quickstart/www/assessment/index.html

```diff
@@ -0,0 +1,2 @@
+<!-- Redirect to ../index.html -->
```

Comment:
Needed?

Reply:
Probably not. We'll look at removing it.

Reply:
Removed.